SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -2,12 +2,41 @@
 //  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
-#import "UIColor+OWS.h"
 #import "OWSMath.h"
+#import "Theme.h"
+#import "UIColor+OWS.h"
 #import <SignalServiceKit/Cryptography.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+
+@interface OWSConversationColors ()
+
+@property (nonatomic) UIColor *defaultColor;
```

Comment:
Maybe `primaryColor` or `mainColor` would be better than `defaultColor` so as not to conflict with the other usage of `default` as in `ows_defaultConversationColors`